docs/netlify/functions/feedback.ts

```diff
@@ -0,0 +1,101 @@
+import type { Handler, HandlerEvent, HandlerContext } from '@netlify/functions';
+
+const handler: Handler = async (event: HandlerEvent, context: HandlerContext) => {
+  // Only allow POST requests
+  if (event.httpMethod !== 'POST') {
+    return {
+      statusCode: 405,
+      body: JSON.stringify({ error: 'Method not allowed' })
+    };
+  }
+
+  try {
+    const data = JSON.parse(event.body || '{}');
+    const { type, page, message } = data;
+
+    // Get GitHub token from environment variable
+    const GITHUB_TOKEN = process.env.GITHUB_TOKEN;
+    const GITHUB_REPO = 'interledger/documentation-feedback';
```

Comment:
Just to check is this correct? Or should it be "interledger/open-payments-docs-feedback"